authors/apps/articles/import_statements.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import generics
```

Comment:
@armstrongsouljah this file is unnecessary, please undo this change and leave the imports at the top of the files as they were.

Reply:
working on it.